lib/constants.js

```diff
@@ -0,0 +1,31 @@
+export const VERIFICATION_STATUS = {
+    NOT_SIGNED: 0,
+    SIGNED_AND_VALID: 1,
+    SIGNED_AND_INVALID: 2
+};
+
+export const SRP_LEN = 2048;
+
+export const AUTH_FALLBACK_VERSION = 2;
+export const AUTH_VERSION = 4;
+
+export const MAX_VALUE_ITERATIONS = 1000;
+
+export const SRP_MODULUS_KEY = `
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xjMEXAHLgxYJKwYBBAHaRw8BAQdAFurWXXwjTemqjD7CXjXVyKf0of7n9Ctm
+L8v9enkzggHNEnByb3RvbkBzcnAubW9kdWx1c8J3BBAWCgApBQJcAcuDBgsJ
+BwgDAgkQNQWFxOlRjyYEFQgKAgMWAgECGQECGwMCHgEAAPGRAP9sauJsW12U
+MnTQUZpsbJb53d0Wv55mZIIiJL2XulpWPQD/V6NglBd96lZKBmInSXX/kXat
+Sv+y0io+LR8i2+jV+AbOOARcAcuDEgorBgEEAZdVAQUBAQdAeJHUz1c9+KfE
+kSIgcBRE3WuXC4oj5a2/U3oASExGDW4DAQgHwmEEGBYIABMFAlwBy4MJEDUF
+hcTpUY8mAhsMAAD/XQD8DxNI6E78meodQI+wLsrKLeHn32iLvUqJbVDhfWSU
+WO4BAMcm1u02t4VKw++ttECPt+HUgPUq5pqQWe5Q2cW4TMsE
+=Y4Mw
+-----END PGP PUBLIC KEY BLOCK-----
+`;
+
+// Version 2 of bcrypt with 10 rounds.
```

Comment:
```suggestion
// Version 2 of bcrypt with 2**10 rounds.
```
Just so that casual readers don't start writing blog posts about how insecure we are 🙂